packages/ember-metal/lib/tags.js

```diff
@@ -1,5 +1,7 @@
 import { meta as metaFor } from './meta';
 import require, { has } from 'require';
+import run from './run_loop';
+import { schedulerRegistrar } from 'ember-glimmer/renderer';
```

Comment:
I did that at first, but it doesn't feel right b/c `schedulerRegistrar` is so highly related to the glimmer renderer.